engine/admin/resource_constraints.md

```diff
@@ -38,16 +38,22 @@ for [Memory Resource Controller](https://www.kernel.org/doc/Documentation/cgroup
 
 ### `--memory-swap` details
 
-- If unset, and `--memory` is set, the container can use twice as much swap
+- A limit value equal to memory plus swap. Must be used with `-m (--memory)` flag.
+      The limit should always be larger than `-m (--memory)` value. By default, it is
+      set to double the value of --memory.
```

Comment:
How can this be true and at the same time the default can be `-1` (as stated below), which means unlimited swap memory?